ompi/mca/coll/libnbc/libdict/hb_tree.c

```diff
@@ -823,14 +799,14 @@ hb_itor_nextn(hb_itor *itor, unsigned count)
 	RETVALID(itor);
 }
 
-static int
-hb_itor_prevn(hb_itor *itor, unsigned count)
+int
+ompi_mca_coll_libnbc_hb_itor_prevn(hb_itor *itor, unsigned count)
```

Comment:
Same here: why remove the `static` qualifier?

Reply:
I'm not sure which direction I should proceed here. 

There are a number of functions in dict.c and hb_tree.c that are not used anywhere. If I mark them static then the compiler is able to detect that they are unused and issue a warning. I asked @gpaulsen about this and he considers libdict to be a library that could be used elsewhere, and that these functions were part of it's API tthat currently is not used. 

If I remove static, then the original names violate OpenMPI rules for external symbol names, so I prefix the symbols with ompi_mca_coll_libnbc_. 

If libdict is supposed to be a library for general use then the symbols should be renamed, and not marked static.

Reply:
You can mark them `static inline`. At least in GCC (and I assume Clang) it should suppress the warning: https://stackoverflow.com/a/47542838/16791592